includes/class-wc-stripe-api.php

```diff
@@ -272,6 +272,9 @@ public static function retrieve( $api ) {
 			// We delete the transient for the account data to trigger the not-connected UI in the admin dashboard.
 			delete_transient( WC_Stripe_Mode::is_test() ? WC_Stripe_Account::TEST_ACCOUNT_OPTION : WC_Stripe_Account::LIVE_ACCOUNT_OPTION );
 
+			// Stripe redacts API keys in the response.
+			WC_Stripe_Logger::log( "Error: GET {$api} returned a 401 " . print_r( $response, true ) );
+
```

Comment:
Should we log to the main PHP error log as well? If our logging is disabled, this will be a no-op.

Reply:
Per this internal discussion, we're going to keep this to our plugin log for now. p1747830648104869/1747828872.340459-slack-C08SUDB2YKC